src/verb/internal_focus.rs

```diff
@@ -122,7 +124,13 @@ fn path_from_input(
             // The given path may be relative hence the need for the
             // state's selection
             // (we assume a check before ensured it doesn't need an input)
-            path::path_from(base_path, PathAnchor::Unspecified, verb_arg)
+            let path_builder = ExecutionStringBuilder::with_invocation(
```

Comment:
Here we use the `ExecutionStringBuilder` so that we can use the `{...}` substitutions in arguments to the `focus` verb.